app/addons/fauxton/navigation/stores.js

```diff
@@ -94,8 +107,7 @@ Stores.NavBarStore = FauxtonAPI.Store.extend({
   },
 
   toggleMenu () {
-    app.utils.localStorageSet(FauxtonAPI.constants.LOCAL_STORAGE.SIDEBAR_MINIMIZED,
-                              !this.isMinimized());
+    this._isMinimized = !this._isMinimized;
```

Comment:
Looks that way, yeah.  @justin-mcdavid-ibm: did you discuss this with @robertkowalski during the initial design phase for this?